server/zanata-frontend/src/app/editor/components/MTSuggestionsButton/index.js

```diff
@@ -0,0 +1,31 @@
+import Button from 'antd/lib/button'
+import 'antd/lib/button/style/css'
+import Tooltip from 'antd/lib/tooltip'
+import 'antd/lib/tooltip/style/css'
+import React from 'react'
+import * as PropTypes from 'prop-types'
+
+class MTSuggestionsButton extends React.Component {
+  static propTypes = {
+    backendId: PropTypes.string.isRequired
+  }
+  render () {
+    const backendId = 'Google'
```

Comment:
Should this say backendId = this.props.backendId?

Reply:
Aha, it should. Shall fix.

Reply:
If I make that change the props used in storybook no longer work. ie. both stories show the google translate image. I think this may have to do with a default prop being needed, but I am unsure. @efloden checked my code and said it looked fine so perhaps he can explain why it doesn't work if I make your requested change.